app/containers/Editor.js

```diff
@@ -0,0 +1,59 @@
+var React = require('react');
+var ReactDOM = require('react-dom');
+var styles = require('../styles');
+var ContentEditable = require('react-contenteditable').default;
+
+var Editor = React.createClass({
+
+	getInitialState: function () {
+		var drinkObject= this.props.object;
+		return {
+			html: drinkObject.name,
+			html2: drinkObject.ingredients,
+			drinkNumber: drinkObject.drinkNumber,
+			drinkList: JSON.parse(localStorage.getItem('drinkList')),
+		};
+	},
+
+	handleNameChange: function (e) {
```

Comment:
an easy way to handle state changes with one function for multiple values [Stack Overflow Answer](http://stackoverflow.com/a/21031832)